src/TLuaInterpreter.h

```diff
@@ -409,14 +414,6 @@ class TLuaInterpreter : public QThread
     static std::map<lua_State*, Host*> luaInterpreterMap;
     QMap<QNetworkReply*, QString> downloadMap;
 
-public slots:
-
-    void slot_replyFinished( QNetworkReply * );
-    void slotPurge();
-    void slotDeleteSender();
-
-private:
-
     lua_State* pGlobalLua;
     TLuaMainThread* mpLuaSessionThread;
```

Comment:
As `TLuaInterpreter::startLuaSessionInterpreter()` was not used (unless you have recently done something with it) `mpLuaSessionThread` is also cruft left over from old code and can, _I think_ be excised.